be/src/vec/exec/scan/vscanner.cpp

```diff
@@ -67,12 +67,12 @@ Status VScanner::get_block(RuntimeState* state, Block* block, bool* eof) {
             }
 
             // 2. Filter the output block finally.
-            {
+            if (!_enable_remaining_expr_pushdown) {
                 SCOPED_TIMER(_parent->_filter_timer);
```

Comment:
Maybe a problem here. Runtime filter comes after scanner start. Then runtime filter will never take affect.

Reply:
Copy `_vconjunct_ctx_ptr` to `_common_vexpr_ctxs_pushdown` and push down, mark `_vconjunct_ctx_ptr` as stale